src/ray/protobuf/gcs.proto

```diff
@@ -444,27 +444,31 @@ message ResourcesData {
   bytes node_id = 1;
   // Resource capacity currently available on this node manager.
   map<string, double> resources_available = 2;
-  // Indicates whether available resources is changed. Only used when light
-  // heartbeat enabled.
-  bool resources_available_changed = 3;
+  // Whether this node has object pulls queued. This can happen if
+  // the node has more pull requests than available object store
+  // memory. This is a proxy for available object store memory.
+  bool object_pulls_queued = 3;
```

Comment:
It's unfortunate we can't easily write a C++ unit test to verify this value is set correctly.

Could we test it in Python via the cluster resources API, or by adding it to memory stats (memory_summary())?

Reply:
Yeah, it's a little tricky because we don't have a way to control timing for object pulls, but I'll see what I can do.

Reply:
I decided against doing this because it requires changing a whole bunch of GCS protos and I think it will be confusing to users anyway. Is just C++ test for the pull manager okay?

Reply:
How about just adding it to memory summary then? I think this test would work (similar to the other tests in test_plasma_unlimited):
```
launch a whole bunch of tasks with huge input args that just sleep
call internal_api.memory_summary(), check "num nodes at capacity value"
```